sources/api/pluto/src/main.rs

```diff
@@ -452,6 +499,8 @@ async fn run() -> Result<()> {
             .context(error::SetFailureSnafu)?;
     }
 
+    // unset the AWS_CONFIG_FILE_ENV_VAR environment variable which was added via `set_aws_config`.
+    env::remove_var(AWS_CONFIG_FILE_ENV_VAR);
```

Comment:
Here, we could also remove `AWS_CONFIG_FILE_PATH`, but I left this as is to see if there are other opinions

Reply:
Why does it have to be removed? If `pluto` is a `oneshot` service that runs on its own, do we really need to clean up the environment?

Reply:
Yea, good point. I included this as a "better-safe-than-sorry" measure, but it's not necessary given `pluto` is a [oneshot](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/a10182f974abd19de803bb7a324b5a0bdeab4ced/packages/os/pluto.service#L17) service